frontend/src/components/query/query-builder/steps/order/order-case-tag.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+import { OrderColumn } from "@/definitions";
+import { Icon, IconName, Tag, Text } from "@blueprintjs/core";
+import SquareIcon, { SquareIconSize } from "@/components/icon/square-icon";
+import { useTable } from "@/data/use-tables";
+
+interface OrderCaseTagProps {
+  orderCase: OrderColumn;
+  index: number;
+  removeable: boolean;
+  removeCase: (index: number) => void;
+}
+
+export default function OrderCaseTag({
+  orderCase,
+  index,
+  removeable,
+  removeCase,
+}: OrderCaseTagProps) {
+  const {
+    data: table,
+    isLoading: isLoadingTable,
+    error: tableError,
+  } = useTable(orderCase.column.table);
+
+  if (tableError && orderCase.column.table !== "aggregate") {
```

Comment:
Why if not aggregate specifically? Prob add a comment for such a specific condition

Reply:
For aggregate columns, we use "aggregate" in the table field (instead of the ID of the origin table in the case of a a normal column). Added a comment explaining this